src/MathCaptcha.php

```diff
@@ -38,7 +50,9 @@ public function label()
     public function input(array $attributes = [])
     {
         $attributes['type'] = 'text';
+        $attributes['id'] = 'mathcaptcha';
```

Comment:
Just some tips: Always check out the latest "master" branch version. You've done your commit on the 1.0.2 tag, while there has been a 1.0.3 version.